test/test.js

```diff
@@ -1,100 +1,132 @@
-const should = require('chai').should()
-const Nightmare = require('nightmare')
-const { percySnapshot } = require('@percy/nightmare')
+const should = require('chai').should();
+const Nightmare = require('nightmare');
+const httpServer = require('http-server');
+const percySnapshot = require('@percy/nightmare');
 
-const TEST_URL = "http://localhost:8000"
+const PORT = process.env.PORT_NUMBER || 8000;
+const TEST_URL = `http://localhost:${PORT}`;
+
+// xvfb wrapper
+const xvfb = {
+  start: () =>
+    new Promise((resolve) => {
+      xvfb.instance = new (require('xvfb'))();
+      xvfb.instance.start(() => resolve());
+    }),
+
+  stop: () =>
+    new Promise((resolve) => {
+      xvfb.instance.stop(() => resolve());
+    })
+};
 
 describe('TodoMVC', function () {
-  this.timeout('10s')
+  let server, nightmare;
+
+  this.timeout('10s');
```

Comment:
Is this timeout still necessary?

Reply:
I think so, it's in their examples: https://github.com/segmentio/nightmare#examples

Reply:
🤷